napari/viewer.py

```diff
@@ -105,7 +105,7 @@ def screenshot(
         path : str
             Filename for saving screenshot image.
         size : tuple (int, int)
-            Size (resolution) of the screenshot. By default, the currently displayed size.
+            Size (resolution Height x Width) of the screenshot. By default, the currently displayed size.
```

Comment:
```suggestion
            Size (resolution height x width) of the screenshot. By default, the currently displayed size.
```

Reply:
Resolved with [ca18ef1](https://github.com/napari/napari/pull/5432/commits/ca18ef1c0c58e4cfe0e052ce2e12b7f660cff101) due to not being able to commit the suggestion